its/src/test/kotlin/org/sonarlint/intellij/its/utils/SettingsUtils.kt

```diff
@@ -81,6 +81,18 @@ class SettingsUtils {
             }
         }
 
+        fun toggleRule() {
+            sonarLintGlobalSettings {
+                findText("Rules").click()
+                searchRule("java:S139")
+                findText("Comments should not be located at the end of lines of code").doubleClick()
+                Pause.pause(5000)
```

Comment:
I understand why we need this pause, but have you tried with something more dynamic? (waiting for the `apply` button to be enabled for instance, if that's possible)